test/e2e/scheduling/resource_quota.go

```diff
@@ -365,7 +365,9 @@ var _ = SIGDescribe("ResourceQuota", func() {
 	})
 
 	It("should create a ResourceQuota and capture the life of a configMap.", func() {
-		found, unchanged := 0, 0
+		cms, _ := f.ClientSet.CoreV1().ConfigMaps(f.Namespace.Name).List(metav1.ListOptions{})
+		found := len(cms.Items)
```

Comment:
I thought we created a new namespace for each test, so if you're trying to figure out what's going on here, it might be useful to print any items you do find (I wouldn't expect any!)